docs/tools/vdb_table/data/rockset.json

```diff
@@ -109,44 +109,44 @@
   },
   "pricing": {
     "value": "",
-    "source_url": "",
+    "source_url": "https://rockset.com/pricing/",
     "comment": ""
   },
   "in_process": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "multi_tenancy": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "disk_index": {
-    "support": "",
+    "support": "full",
```

Comment:
same as above. Please provide reference to docs/example notebook in source_url. It is a requirement for any new support claim.